Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/README.md

```diff
@@ -1,386 +1,218 @@
+- Send notifications to external users.
+- Send an email asking for a response to be returned as part of a playbook. See [Receiving an email reply](https://xsoar.pan.dev/docs/reference/scripts/email-ask-user)
+
+## Configure EWS v2 on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for EWS v2.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Email address | True |
+    | Password | True |
+    | Email address from which to fetch incidents | True |
+    | Name of the folder from which to fetch incidents (supports Exchange Folder ID and sub-folders e.g. Inbox/Phishing) | True |
+    | Public Folder | False |
+    | Has impersonation rights | False |
+    | Use system proxy settings | False |
+    | Fetch incidents | False |
+    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | False |
+    | Mark fetched emails as read | False |
+    | Incident type | False |
+    | ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉<br/>‎                                           Manual Mode<br/>Exchange Server Hostname or IP address | False |
+    | DOMAIN\USERNAME (e.g. DEMISTO.INT\admin) | False |
+    | Exchange Server Version (On-Premise only. Supported versions: 2007, 2010, 2010_SP2, 2013, and 2016) | False |
+    | Trust any certificate (not secure) | False |
+    | ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉<br/>‎                                         Advanced Mode<br/>Override Authentication Type (NTLM, Basic, or Digest). | False |
+    | Timeout (in seconds) for HTTP requests to Exchange Server | False |
+    | Max incidents per fetch | False |
+    | Run as a separate process (protects against memory depletion) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Fetched Incidents Data
+The integration imports email messages from the destination folder in the target mailbox as incidents. If the message contains any attachments, they are uploaded to the War Room as files. If the attachment is an email, Cortex XSOAR fetches information about the attached email and downloads all of its attachments (if there are any) as files.
+
+To use Fetch incidents, configure a new instance and select the ```Fetches incidents``` option in the instance settings.
+
+IMPORTANT: The initial fetch interval is the previous 10 minutes. If no emails were fetched before from the destination folder- all emails from 10 minutes prior to the instance configuration and up to the current time will be fetched. Additionally moving messages manually to the destination folder will not trigger fetch incident. Define rules on phishing/target mailbox instead of moving messages manually.
```

Comment:
```suggestion
**IMPORTANT:** The initial fetch interval is the previous 10 minutes. If no emails were fetched before from the destination folder, all emails from 10 minutes prior to the instance configuration and up to the current time will be fetched. Additionally, moving messages manually to the destination folder will not trigger a fetch incident. Define rules on phishing/target mailbox instead of moving messages manually.
```